net/init-metrics.sh

```diff
@@ -83,11 +83,16 @@ else
   query "CREATE DATABASE \"$netBasename\""
   query "ALTER RETENTION POLICY autogen ON \"$netBasename\" DURATION 7d"
   query "GRANT READ ON \"$netBasename\" TO \"ro\""
-  query "GRANT WRITE ON \"$netBasename\" TO \"scratch_writer\""
+  query "GRANT WRITE ON \"$netBasename\" TO \"${username}\""
 
-  SOLANA_METRICS_CONFIG="host=$host,db=$netBasename,u=scratch_writer,p=topsecret"
+  SOLANA_METRICS_CONFIG="host=$host,db=$netBasename,u=${username},p=${password}"
+  set +x
+  echo "Consider adding \`SOLANA_METRICS_CONFIG\` to your shell environment for future use:"
+  echo "export SOLANA_METRICS_CONFIG=\"$SOLANA_METRICS_CONFIG\""
```

Comment:
do we still want this?

Reply:
technically it was there previously, just was not explicit.

Reply:
I don't understand.
Where do we add `SOLANA_METRICS_CONFIG` to the shell environment today?
It is only added to `config/config`.

Reply:
can you also please clarify here. thank you.

Reply:
Well it was implied by the -e flag of the very same script.

Reply:
Can you please clarify where in the script this is implied? and how is that used?

The problem here is that, we shouldn't encourage people to use/change their env variables. Doing so we will ultimately end-up in the situation that something does not work on one machine but it is not reproducible on another machine and we end up wasting a lot of time debugging.

Reply:
scripts/configure_metrics.sh first tries the env variable, and if it can not find it then reads from config file. 
Also we do not encourage, we ask user to consider if they want to add it.

Reply:
As chatted on slack, I did start a gce cluster from my machine (without using docker), and there was no solana related variables in my env before or after I ran the scripts. no env variables in my local shell was changed either.

I still think the best is to remove these 2 lines not to confuse new users.